tests/testthat/test-5transformCounts.R

```diff
@@ -216,18 +198,8 @@ test_that("transformCounts", {
         expect_equal(as.matrix(assays(mia::transformCounts(tse, method = "rank"))$rank),
                      as.matrix(assays(mia::transformSamples(tse, method = "rank"))$rank))
         
-        # Tests transformFeatures, calculates rank
-        # Calculates rank
-        tse_rank <- transformFeatures(tse, method = "rank")
-        for( i in c(1:10) ){
-            # Gets rows from 'rank' table
-            ranks <- assay(tse_rank, "rank")[i,]
-            # Gets rows from 'counts' table, and calculates ranks
-            counts_compare <- assay(tse_rank, "counts")[i,]
-            ranks_compare <- rank(counts_compare, na.last = "keep", ties.method = "first")
-            # Expect that they are equal
-            expect_equal(ranks, ranks_compare)
-        }
+        # Tests transformFeatures, tries to calculate rank. Should be an error.
+        expect_error(mia::transformFeatures(tse, method = "rank"))
         
```

Comment:
Why was this removed? I think it is an important test.

Reply:
`transformFeatures` does not provide `rank `anymore. But `rank `transformation for samples (`transformSamples`) are still tested